backend/x11/backend.c

```diff
@@ -428,7 +425,6 @@ struct wlr_backend *wlr_x11_backend_create(struct wl_display *display,
 error_event:
 	wl_event_source_remove(x11->event_source);
 error_x11:
-	xcb_disconnect(x11->xcb_conn);
```

Comment:
Should we call `xcb_disconnect` if `XOpenDisplay` failed? (around line 375 in new file, github doesn't let me comment there)

Reply:
Or actually it's all fine, don't mind me I'm tired. Thanks for looking it up & the patch.